toolkit/FileManager.hpp

```diff
@@ -0,0 +1,48 @@
+#ifndef DARWIN_FILEMANAGER_HPP
+#define DARWIN_FILEMANAGER_HPP
+
+#include <mutex>
+#include <thread>
+#include <fstream>
+
+#include "tsl/hopscotch_map.h"
+
+/// \namespace darwin
+namespace darwin {
+    /// \namespace toolkit
+    namespace toolkit {
+
+        class FileManager {
+
+        public:
+            /// FileManager constructor
+            /// \param file the file's path to manage
+            /// \param append if we append data to the file
+            explicit FileManager(std::string& file, bool app=true);
+            ~FileManager();
+
+            /// Open the file
+            /// \return true if successful, else false
+            bool Open();
+
+            /// Write in the file
+            /// \return true if successful, else false
+            bool Write(std::string s);
```

Comment:
```suggestion
            bool Write(std::string& s);
```
Cf. previous comment